src/core/engine.js

```diff
@@ -227,6 +230,26 @@ function engineFactory({
             return !!maths.degree;
         },
 
+        /**
+         * Sets the engine to compute the expression instantaneously (`true`) or not ('false').
+         * @param {boolean} mode - The state of the instant mode.
+         * @returns {calculator}
+         * @fires configure
+         */
+        setInstantMode(mode = true) {
+            instant = mode;
```

Comment:
Well, I would have done so if the `instant` variable didn't already exist in the higher scope.
Here, I need to do two things:
- set the `instant` variable so that the state is changed
- dispatch the info through an event

I cannot use the same name for the state variable and for the parameter, it would conflict.

Reply:
:+1: